### PR TITLE
Clean up NodeJS guidance

### PR DESCRIPTION
**Pull request by injms**
Minor formatting tweaks to the NodeJS at GDS guide - the main one being to make the code examples conform to StandardJS, as that's the convention that should be used when writing code.

Note - I haven't touched the section on whether NPM is the tool of choice. I believe that Yarn is used in certain parts of the organisation, but am unsure whether that's tied to Ruby on Rails or not. I think that's a can of worms that is best resolved in another pull request as it's a fairly major change.

### CODE REVIEW

File: source/manuals/programming-languages/nodejs/index.html.md.erb [hunk 1]
```diff
@@ -445,21 +438,20 @@ for (let i = 0; i < 10; i += 1) {
 }
 
 // Over:
-for (let i=0,j=0;i<10 && j<10;j++,i=(j==10)?i+1:i,j=(j==10)?j=0:j,console.log(i,j)){}
+for (let i = 0, j = 0; i < 10 && j < 10; j++, i = (j == 10) ? i + 1 : i, j = (j == 10) ? j = 0 : j, console.log(i, j)) {}
```
Comment (tombye) [inline source/manuals/programming-languages/nodejs/index.html.md.erb:441]: At a glance it seems a bit odd to add spacing to an example of code written to take up less space. Do you think it's needed here?

**Review by tombye — COMMENTED** [review]
The majority of these changes look fine to me, I only made one comment and it's not a blocker if you feel the change it references should happen.

File: source/manuals/programming-languages/nodejs/index.html.md.erb (hunk 1 above)
Comment (injms) [reply source/manuals/programming-languages/nodejs/index.html.md.erb]: Good point - I've removed this change.

File: source/manuals/programming-languages/nodejs/index.html.md.erb [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Using Node.js at GDS
-last_reviewed_on: 2021-07-02
+last_reviewed_on: 2021-03-15
```
Comment (querkmachine) [inline source/manuals/programming-languages/nodejs/index.html.md.erb:3]:
```suggestion
last_reviewed_on: 2022-03-15
```

Comment (injms) [reply source/manuals/programming-languages/nodejs/index.html.md.erb:3]: Good spot - thank you!

**Review by tombye — APPROVED** [review]
Thanks for doing the changes. Looks good to me now 👍🏻

**Pull request merged**